tests/playwright/src/model/pages/kube-context-page.ts

```diff
@@ -0,0 +1,95 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Locator, Page } from '@playwright/test';
+import { expect as playExpect } from '@playwright/test';
+
+import { SettingsPage } from './settings-page';
+
+export class KubeContextPage extends SettingsPage {
```

Comment:
Add any new page/class from the playwright tests into index.ts. Otherwise the class wont be available when `@podman-desktop/tests-playwright` npm package is used outside of the podman-desktop repo.